LinkedList.java

```diff
@@ -1,30 +0,0 @@
-
-public class LinkedList {
-	private Link first;
-	
-	public LinkedList () {
-		first = null;
-	}
-	
-	public boolean isEmpty () {
-		return (first == null);
-	}
-	
-	public void insert (Object data) {
```

Comment:
It is worth explaining where this is inserting. You have to read the code in the method to find out if this is inserting in the front or the back.